src/main/resources/templates/register.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <link href="bootstrap/css/bootstrap.min.css"
+              th:href="@{bootstrap/css/bootstrap.min.css}" rel="stylesheet" media="screen"/>
+        <meta charset="UTF-8"/>
+        <title>Register</title>
+    </head>
+    <style>
+        body {
```

Comment:
We need to decide whether we want the h1 tag to have this styling throughout the website.